src/SassBuilder.php

```diff
@@ -90,9 +90,11 @@ public function setOutput(SymfonyStyle $output): void
     /**
      * @internal
      */
-    public static function guessCssNameFromSassFile(string $sassFile, string $outputDirectory): string
+    public static function guessCssNameFromSassFile(string $sassFile, string|int $fileName, string $outputDirectory): string
```

Comment:
Hm, this method is public so it might be considered as a BC break too.

I suppose if we move this arg to the end and make it null by default - it may help with avoiding BC break.